cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -816,6 +825,34 @@ void scheduleFilteredBanners() {
     }
   }
 
+  private boolean checkIsEveryTrigger(Banner banner, boolean isEveryTrigger) {
+    String bannerEventId = null;
+    boolean isConditionTrue = false;
+    if (isEveryTrigger && banner.getTriggerType() == BannerTriggerType.Conditions) {
```

Comment:
remove this if here
and do this at the beginning 
if(!isEveryTrigger || banner.getTriggerType() != BannerTriggerType.Conditions){
return true;
}